### ISSUE TITLE
Relation has been removed

### ISSUE THREAD

**Issue by dkuppitz** [thinkaurelius/titan#225]
Store a vertex in a variable (in Gremlin shell), then commit the current transaction. After commit you won't be able to use the variable for mutating queries.

```
gremlin> g = TitanFactory.open('/tmp/test')
==>titangraph[local:/tmp/test]
gremlin> g.V
gremlin> v1 = g.addVertex()
==>v[4]
gremlin> v2 = g.addVertex()
==>v[8]
gremlin> g.addEdge(null,v1,v2,'knows')
==>e[d-4-2F0LaTPQAK][4-knows->8]
gremlin> g.commit()
==>null
gremlin> v3 = g.addVertex()
==>v[12]
gremlin> g.addEdge(null,v1,v3,'knows')
Relation has been removed
Display stack trace? [yN] N
```

This should not happen (see [discussion](https://groups.google.com/forum/?fromgroups=#!topic/aureliusgraphs/T9qrLnhWuys)).

**Comment by okram** [comment]
Ha. I just reproduced this problem and was like: "I saw this recently?! What is this again..." To repeat, g.addEdge() doesn't work when one of the vertex variables is expired out of the transaction (v.addEdge() works though). This is a real bummer cause now Frames is unusable around @Adjacency. We can't seem to get the work around of re-getting the vertex in the transaction either..... The hack we are using to make this work for Frames is to reframe the vertex on each TX commit and then refreshes' the variable.
